src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -1300,6 +1299,7 @@ private BoundExpression MakeLiftedUserDefinedConversionConsequence(BoundCall cal
             Debug.Assert(rewrittenOperand != null);
             Debug.Assert((object)rewrittenType != null);
             Debug.Assert(rewrittenOperand.Type is { });
+            Debug.Assert(!_compilation.Assembly.RuntimeSupportsNumericIntPtr);
```

Comment:
>!_compilation.Assembly.RuntimeSupportsNumericIntPtr [](http://example.com/codeflow?start=25&length=51)

`_compilation.ShouldEmitNativeIntegerAttributes()`?

Reply:
I prefer the current logic because this method doesn't relate to attributes.